src/components/views/elements/AddressTile.tsx

```diff
@@ -49,11 +49,10 @@ export default class AddressTile extends React.Component {
         if (isMatrixAddress && address.avatarMxc) {
             imgUrls.push(mediaFromMxc(address.avatarMxc).getSquareThumbnailHttp(25));
         } else if (address.addressType === 'email') {
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
             imgUrls.push(require("../../../../res/img/icon-email-user.svg"));
```

Comment:
I am not very sure what would be the correct fix

Reply:
I believe you could do something like the following:

```ts
import EmailIconUser = require("../../../../res/img/icon-email-user.svg");

// ...

imgUrls.push(EmailIconUser);
```

This requires declaring an `svg` module, by creating a new file under `src/@types` (I **think** this is the correct location for this?):

```ts
# src/@types/svg.d.ts

declare module "*.svg" {
    const content: any;
    export default content;
}
```

Reply:
the import itself should work, ts would treat it as `any` - the type definition would be a plus. Though avoid `import = require` - a bare import should work fine

Reply:
Without the type definition I get the following error when building:

```shell
Cannot find module '../../../../res/img/icon-email-user.svg' or its corresponding type declarations.
```

So it appears it is required? I'm a TypeScript beginner so I could definitely be wrong. 

> Though avoid `import = require` - a bare import should work fine

👍 I edited my snippet accordingly

Reply:
Thanks both for the help with this!